app/build.gradle

```diff
@@ -40,8 +52,7 @@ android {
         compose true
     }
     composeOptions {
-        kotlinCompilerExtensionVersion compose_version
-        kotlinCompilerVersion '1.5.10'
+        kotlinCompilerExtensionVersion "1.1.0-rc02"
```

Comment:
The current version of compose is weird in that we need to use a different compiler version than the one used in the actual dependencies.

Next compose versions might go back to usual, so you could go back to `compose_version` if that's the case.